src/main/java/seedu/taassist/model/moduleclass/ModuleClass.java

```diff
@@ -70,13 +82,12 @@ public String getClassName() {
      * if modification is attempted.
      */
     public List<Session> getSessions() {
```

Comment:
I think `List<Session>` should be `ObservableList<Session>`. Return type should not limit functionality. :thinking: 

Can you also check in other places where observable list is returned as list?

Reply:
ok, other parts of the code should also be fixed